src/gui/properties/peerlistsortmodel.h

```diff
@@ -31,29 +31,61 @@
 #ifndef PEERLISTSORTMODEL_H
 #define PEERLISTSORTMODEL_H
 
-#include <QStringList>
 #include <QSortFilterProxyModel>
+
 #include "peerlistdelegate.h"
 
-class PeerListSortModel : public QSortFilterProxyModel {
-  Q_OBJECT
+class PeerListSortModel: public QSortFilterProxyModel
+{
+    Q_OBJECT
```

Comment:
Yes, but reflection for this class is not used, that is why I thought that `Q_OBJECT` can be omitted perhaps.